src/contrib/cluster/Akka.DistributedData/Serialization/OtherMessageComparer.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Akka.DistributedData.Serialization.Proto.Msg;
+
+namespace Akka.DistributedData.Serialization
+{
+    internal class OtherMessageComparer : IComparer<OtherMessage>
```

Comment:
We should make this `static` and have a `readonly` singleton here, but we can easily do that in an update

Reply:
Ah, you already did that inside the serializer